resources/install.rb

```diff
@@ -7,6 +7,8 @@
 property :influxdb_key, kind_of: String, default: 'https://repos.influxdata.com/influxdb.key'
 property :install_version, [String, nil], default: nil
 property :install_type, String, default: 'package'
+property :package_name, String, name_property: true
```

Comment:
can we make `influxdb` the default?

Reply:
Yes we can, but is it necessary? It also has `:name_property: true`, which means value for :package_name will be equal to the 'name' of the resource block.
```
# typical usage
influxdb_install 'influxdb' do
  action [:install]
end

# shortest 
influxdb_install 'abrakadabra'
```
When we define resource in recipe, we always assign it a unique name (influxdb or abrakadabra), so even if we define default name it will be used never. If it's never used - then no need to define it.
Makes sense ?

Reply:
Thanks for the clarification @anatolijd.  I think this seems reasonable, although currently it's a little confusing since you haven't added this variable to the package `install_type`.  If we have this property and it's called `package_name`, it should probably apply to packages. :)

Reply:
Here:  https://github.com/bdangit/chef-influxdb/pull/151/files/036b35434795fa23ea19272cfe0b27859455a88b#diff-ee0c699013153b738cf7637641d5e110L48

Reply:
Changes have been made to installation_type: 'file' only.
To the time of writing, there is no official repository for Influxdb Enterpise, the only recommended way to install/upgrade enterprise packages is to download individual files (.deb .rpm).

Also, it is typical to install both influxdb-data and influxdb-meta at the same node
```
influxdb_install 'influxdb-meta' do
...
end

influxdb_install 'influxdb-data' do
...
end

```
which means nested ```apt_repository 'influxdb'``` resource will be defined twice and chef-client will complain. I could change it ```apt_repository "#{package_name}"``` name too, but it is a bit silly to clone repositories per individual package.

It was just too much, I decided to stay with  installation_type: 'file' only :)

Reply:
Makes sense @anatolijd, the only potential problem I see is that users may not know that the `resource_name` is now very picky and by merging this change, it really throws this expectation out the door.

But then again, I'm not sure if users have just done this 

```
influxdb_install
```

without a property_name.

In that case, I'm okay with this.